pretyping/cases.ml

```diff
@@ -128,7 +128,8 @@ type 'a equation =
       rhs          : 'a rhs;
       alias_stack  : Name.t list;
       eqn_loc      : Loc.t option;
-      used         : bool ref }
+      used         : int ref;
+      catch_all_vars : Id.t CAst.t list ref }
```

Comment:
It's a nit, but I believe you should use `mutable` here rather than `ref` (if you do so you don't need the `!` to deref)